### ISSUE TITLE
Training Setting

### ISSUE THREAD

**Issue by z-jiaming** [hkchengrex/Cutie#10]
Hi, sorry to bother you again.:wave:

I found that Cutie uses an auxiliary loss, and the scaling factor for that loss is 0.1 in the paper, but it is 0.01 in the code.
Am I looking for the wrong settings?

<img width="577" alt="image" src="https://github.com/hkchengrex/Cutie/assets/42411674/440558cd-232a-4f29-bdba-28520aace021">
<img width="201" alt="image" src="https://github.com/hkchengrex/Cutie/assets/42411674/80b7bf05-9570-4bd0-9062-d6ae24988690">

In addition, Cutie does not provide the model pre-trained with BL30K, which your previous works (STCN, XMem) does.
Is this because the new dataset (MOSE, or your MEGA setting) is enough to make up for the past insufficient dataset situation? :hushed: Did you pre-train it with BL30K?

**Comment by hkchengrex** [comment]
1. Right. Thank you for letting us know. I will update the paper in the next revision.
2. We were training on a different server where BL30K is a bit hard to set up because of its size. I do hope the mega setting helps to make it up but I am not sure.

**Comment by z-jiaming** [comment]
Thanks for your reply~~
